examples/map_metadata_test.go

```diff
@@ -9,6 +9,8 @@ import (
 )
 
 func TestGetMapMetadata(t *testing.T) {
+	t.Skip()
```

Comment:
@markus-wa I skip tests that rely on https://radar-overviews.csgo.saiko.tech because it's currently down